src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -295,10 +294,10 @@ private void setupKeyboardShortcuts() {
             if (UNDO_LABEL_CHANGES.match(event)) {
                 ui.triggerNotificationAction();
             }
-            for (KeyCodeCombination key : JUMP_TO_FIRST_ISSUE_KEYS) {
-                if (key.match(event)) {
+            for (Integer nKey:JUMP_TO_NTH_ISSUE_KEYS.keySet()) {
+                if (JUMP_TO_NTH_ISSUE_KEYS.get(nKey).match(event)){
                     event.consume();
-                    listView.selectFirstItem();
+                    listView.selectnthItem(nKey);
```

Comment:
You might want to break out of the loop after this, because the key events are mutually exclusive.